jep/302/README.adoc

```diff
@@ -198,12 +271,14 @@ There are no backwards compatibility concerns related to this proposal.
 
 === Secrets
 
-Versioning secrets is not an issue, as the snapshotting system is going to be local to the running instance.
-As the Git repository data will never be pushed _outside_, no data leak is expected from this side.
+Versioning secrets should not be an issue per se, as the data snapshotting system is designed to be local to the running instance.
+In other words, the Git repository data will never be pushed _outside_ by the _Essentials_ code, so no data leak is normally expected from this side.
+
+But as users may have the unfortunate idea to push that repository elsewhere, not being aware they are leaking all secrets, we will conservatively add `secrets/master.key` to the `.gitignore` file.
```

Comment:
Would be a good place to briefly explain what this file is and why excluding it from the Git repository prevents any straightforward extraction of secrets.

Reply:
I would like to avoid (badly) repeating something that probably ought to be defined in the official docs. But can add a link. 

Something like I guess:

> "``Secret``s are encrypted using the `master.key` files, so we really want to avoid attackers to be able to access both `credentials.xml` and `master.key` at the same time

?